src/Microsoft.AspNetCore.Identity/ITwoFactorSecurityStampValidator.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Identity
+{
+    /// <summary>
+    /// Used to validate the two factor cookies security stamps.
+    /// </summary>
+    public interface ITwoFactorSecurityStampValidator : ISecurityStampValidator
```

Comment:
ICookieValidator and have both of them derive from that instead.  Assuming that's not a breaking change